### ISSUE TITLE
What's the difference between acc and val_acc

### ISSUE THREAD

**Issue by jyoungrok** [aymericdamien/TensorFlow-Examples#65]
When I run the tfLearn example, i get logs printing acc and val_acc
As i understand, val_acc is the accuracy of validation set
However, I am not sure what acc identifies exactly.
Does "acc" mean the accuracy of all of training data of some parts of training data?

And what is validation_batch_size? 
Im confused why batch size is needed...( I know what batch size of training data means)

Thanks in advance
